src/components/views/settings/CrossSigningPanel.js

```diff
@@ -175,7 +179,7 @@ export default class CrossSigningPanel extends React.PureComponent {
         ) {
             bootstrapButton = (
                 <div className="mx_CrossSigningPanel_buttonRow">
-                    <AccessibleButton kind="primary" onClick={this._bootstrapSecureSecretStorage}>
+                    <AccessibleButton kind="primary" onClick={this._onBootstrapClick}>
                         {_t("Bootstrap cross-signing and secret storage")}
                     </AccessibleButton>
                 </div>
```

Comment:
Please add something to the advanced details below to report how the key is stored (the happy path should use text that scans easily for success).

Reply:
Done